mettle/src/tlv.h

```diff
@@ -123,6 +123,7 @@ struct tlv_encryption_ctx {
 
 	uint32_t flag;
 	bool initialized;
+	bool is_weak_key;
```

Comment:
Could this possibly open us up to a negotiation downgrade attack if the RNG itself proves to be predictable?

Reply:
Do you have any scenario in mind you want to share?
The c2 and the agent **never** negotiate the key's strength, mettle will just use the mwtister if mbedtls fails with the platform entropy source and will return the aes_key + is_weak_key = true as TLV package.